test/redisGraphAPITest.js

```diff
@@ -16,8 +16,21 @@ describe("RedisGraphAPI Test", () => {
 		return api.deleteGraph().catch(() => {});
 	});
 
-	it("test bring your client", () => {
-		return new RedisGraph("social", redis.createClient());
+    it("test connection from post and host", async () => {
+        // Assuming this test is running against localhost:6379 with no password.
+        let graph = new RedisGraph("social", "127.0.0.1", 6379, {password:undefined});
```

Comment:
`{password:undefined}` why do you need this?

Reply:
just to show that I can do it.